lib/api/core/plugins/pluginContext.js

```diff
@@ -138,6 +149,15 @@ class PluginContext {
         get: () => curryTrigger(kuzzle, pluginName)
       });
 
+      const throwNotAvailable = (name) => () => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3524)